app/models/originality_report.rb

```diff
@@ -1,5 +1,4 @@
 class OriginalityReport < ApplicationRecord
-  validates :originality_score, :file_id, :tc_id, :submission, presence: true
-  validates :originality_score, inclusion: { in: 0..100, message: 'score must be between 0 and 100' }
```

Comment:
Yes, we don't create the TP in our DB unless Canvas returns a successful response and Canvas does not return a successful response unless the score is nil or between 0..100